### PR TITLE
chore(image): Introduce build and publish + check of the build of the image

### PR DESCRIPTION
**Pull request by benoitf**
Image should go to https://quay.io/repository/che-incubator/che-editor-intellij-community

### CODE REVIEW

**Review by vzhukovskii — APPROVED** [review]

File: .github/workflows/build-publish.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+on:
+    push:
+      paths-ignore:
+      - '**.md'
+  
+jobs:
+  build:
+    env:
+      IMAGE_NAME: quay.io/che-incubator/che-api-sidecar
```
Comment (vzhukovskii) [inline .github/workflows/build-publish.yml:18]: Shouldn't it be che-incubator/che-editor-intellij-community ?

Comment (benoitf) [reply .github/workflows/build-publish.yml]: yes sure, was in the middle of copy/paste :-)

**Comment by benoitf** [comment]
ok, ready to be reviewed :-)

**Comment by benoitf** [comment]
ok, still something wrong on a job, wait :)

**Comment by benoitf** [comment]
ready

**Review by vzhukovskii — APPROVED** [review]
Looks good to me

**Pull request merged**